### ISSUE TITLE
Experiment: can a high weight decay reduce overfitting?

### ISSUE THREAD

**Issue by gonzalobenegas** [Open-Athena/bolinas-dna#23]
## Description

In small datasets, models can start overfitting when trained over multiple epochs. Increasing weight decay could reduce overfitting.
